### PR TITLE
Add Ruby DLP configuration

### PR DESCRIPTION
**Pull request by geigerj**

### CODE REVIEW

File: gapic/api/artman_dlp.yaml [hunk 1]
```diff
@@ -49,3 +49,15 @@ python:
     staging:
       gapic_subpath: generated/python/gapic-google-cloud-dlp-v2beta1
       grpc_subpath: generated/python/proto-google-cloud-dlp-v2beta1
+ruby:
+  gapic_code_dir: ${REPOROOT}/artman/output/ruby/google-cloud-dlp
+  git_repos:
+    ruby:
+      paths:
+        - src: lib/google/cloud/dlp/v1
+          dest: google-cloud-dlp/lib/google/cloud/dlp/v1
```
Comment (swcloud) [inline gapic/api/artman_dlp.yaml]: Do you need to set up the 'test' path mapping for unit tests?

Comment (geigerj) [inline gapic/api/artman_dlp.yaml]: Ah, I'm not familiar with the test path mapping. I followed the pattern for Ruby PubSub here. Could you point me to an example of how the test path should be configured?

Comment (swcloud) [inline gapic/api/artman_dlp.yaml]: https://github.com/googleapis/googleapis/blob/master/gapic/api/artman_dlp.yaml#L42

Comment (lukesneeringer) [inline gapic/api/artman_dlp.yaml]: @geigerj Use Vision or Video as the reference.

Comment (geigerj) [inline gapic/api/artman_dlp.yaml]: Done.

**Comment by geigerj** [comment]
@swcloud Could you take a look at this one, too?

**Pull request merged**